otx/cli/tools/train.py

```diff
@@ -199,6 +241,12 @@ def main():
     assert resultset.performance is not None
     print(resultset.performance)
 
+    if args.save_logs_to:
```

Comment:
save_logs_to is good. Is there any documentation updates for introducing this new args?

Reply:
Not sure where to add this? Do you mean doc documentation? like start_guide.md..?

Reply:
quick start guide.md needed to be updated I think,, I missed to add unlabeled arg there too. Let's update it together later.

Reply:
I meant start guide, it's okay to update later as Soob said